### DISCUSSION TITLE
v2: 在Field的reactions中，判断字段值是否校验通过是用field.valid还是用field.validStatus?

### DISCUSSION THREAD

**Discussion by effapp** [alibaba/formily#1275] [Q&A 2.x] [answered]
v2: 在Field的reactions中，判断字段值是否校验通过是用field.valid还是用field.validStatus?

**Comment by janryWang** [comment, marked as answer]
valid是判断是否合法，validateStatus是校验状态

**Comment by effapp** [comment]
valid是判断是否合法?  请问什么情况下是合法的？我以为校验全部通过了字段值才是合法的。valid和validateStatus是什么关系？判断字段值是否校验通过是用field.valid还是用field.validStatus?
